pages/blog.js

```diff
@@ -1,17 +1,20 @@
-import Link from "next/link";
+import camelcaseKeys from "camelcase-keys";
 
-import camelcaseKeys from 'camelcase-keys';
+import Link from "next/link";
 
-import PostsList from "@/components/blog/posts-list";
+import { getCategories, getPostsData } from "@/lib/api";
 
-import { getPostsData, getCategories } from '@/lib/api'
 import CategoriesWidget from "@/components/blog/categories-widget";
+import PostsList from "@/components/blog/posts-list";
 import SearchWidget from "@/components/blog/search-widget";
 
-export default function Blog({ posts, categories }) {
+export default function Blog({ categories, posts }) {
   return (
     <>
```

Comment:
this is redundant and not needed since you have the `<section>` as a root element

Reply:
There are `<section>`s here

Reply:
ah sorry, my mistake. relooked at the full code to see what Github didn't mark. all good.